core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -284,6 +285,9 @@ public enum Property {
       PropertyType.BOOLEAN, "Enables JVM metrics functionality using Micrometer", "2.1.0"),
   GENERAL_MICROMETER_FACTORY("general.micrometer.factory", "", PropertyType.CLASSNAME,
       "Name of class that implements MeterRegistryFactory", "2.1.0"),
+  GENERAL_PROCESS_BIND_ADDRESS("general.process.bind.addr", "0.0.0.0", PropertyType.STRING,
+      "The local IP address to which this server should bind for sending and receiving network traffic",
+      "3.0.0"),
```

Comment:
Should the bind address include the bind port, or do we still want to keep the port option separate?

Reply:
No, we don't want to specify the port. Each server process has a port property already and some have the ability to do port search.